shared/actions/kbfs.desktop.js

```diff
@@ -297,32 +295,7 @@ function* openWithCurrentMountDir(openPath: string): Saga.SagaGenerator<any, any
   yield Saga.call(_open, resolvedPath)
 }
 
-function* openSaga(action: KBFSGen.OpenPayload): Saga.SagaGenerator<any, any> {
```

Comment:
i'd advise leaving the old actions alone or you'll start breaking the app in production

Reply:
Right now the code is basically a straight port. I'm hoping that we can consolidate to the new code and fix all the bugs in one place.